### DISCUSSION TITLE
Add Neutralinojs to the comparison

### DISCUSSION THREAD

**Discussion by kawpii** [tauri-apps/tauri#3161] [Ideas]
https://github.com/neutralinojs/neutralinojs

**Comment by FabianLars** [comment]
Hmm, i personally don't really see that happening. There are a few problems we'd have to consider first:
- neutralinojs uses the same underlying tech (webview2, webkit2gtk, webkit), so resource usage comparisons will be quite boring.
- at this point we'd have to consider adding other projects like NW.js and NodeGui too and at some point it's just too much. I think it makes sense to just target the largest/most well-known "competitor" and let independent reviewers/projects handle bigger comparisons.
- as far as i know no one in the team is a neutralino expert, so we'd probably have to rely on someone else to add this and more importantly to keep it up to date.
